config/rbac/role.yaml

```diff
@@ -14,6 +14,31 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
FYI, this file is overwritten once make manifests is executed. Add service access needs through comments in one of the controllers. :)

Reply:
yes still thinking of where, plus the `verbs` are still too broad. We probably won't need `watch` for example.

Reply:
I think it would be fine to put it on the main reconcile loop which is the component-version-controller.
WDYT @phoban01 ?